### PR TITLE
[StyleCop] Lua fixes

### PR DESCRIPTION
**Pull request by Randshot**
This fixes all of the refactoring issues in the Lua files.

### CODE REVIEW

**Comment by Dormanil** [comment]
Can confirm, everything runs as it should.

File: Assets/StreamingAssets/LUA/Furniture.lua [hunk 1]
```diff
@@ -184,11 +189,11 @@ function Stockpile_UpdateAction( furniture, deltaTime )
         return
     end
 
-    -- We currently are NOT full, but we don't have a job either.
+    -- We Currently are NOT full, but we don't have a job either.
```
Comment (mikejbrown) [inline Assets/StreamingAssets/LUA/Furniture.lua]: :laughing: search and replace FTW :smile:

**Comment by mikejbrown** [comment]
Am getting the odd
```
chunk_2:(277,1-157): cannot access field ObjectType of userdata<Inventory>
UnityEngine.Debug:LogError(Object)
UberLogger.Logger:ForwardToUnity(Object, LogSeverity, Object, Object[]) (at Assets/UberLogger/UberLogger.cs:494)
UberLogger.Logger:Log(String, Object, LogSeverity, Object, Object[]) (at Assets/UberLogger/UberLogger.cs:452)
Debug:ULogErrorChannel(String, String, Object[]) (at Assets/UberLogger/Debug.cs:176)
LuaUtilities:CallFunction(String, Object[]) (at Assets/Scripts/Utilities/LuaUtilities.cs:44)
FurnitureActions:CallFunctionsWithFurniture(String[], Furniture, Single) (at Assets/Scripts/Models/FurnitureActions.cs:43)
EventAction:Trigger(String, Furniture, Single) (at Assets/Scripts/Models/EventAction.cs:134)
Furniture:Update(Single) (at Assets/Scripts/Models/Furniture.cs:334)
World:Tick(Single) (at Assets/Scripts/Models/World.cs:159)
WorldController:Update() (at Assets/Scripts/Controllers/WorldController.cs:140)
```
followed by
```
NullReferenceException: Object reference not set to an instance of an object
FurnitureActions.CallFunctionsWithFurniture[Furniture] (System.String[] functionNames, .Furniture furn, Single deltaTime) (at Assets/Scripts/Models/FurnitureActions.cs:45)
EventAction.Trigger (System.String actionName, .Furniture target, Single deltaTime) (at Assets/Scripts/Models/EventAction.cs:134)
Furniture.Update (Single deltaTime) (at Assets/Scripts/Models/Furniture.cs:334)
World.Tick (Single deltaTime) (at Assets/Scripts/Models/World.cs:159)
WorldController.Update () (at Assets/Scripts/Controllers/WorldController.cs:140)
```
Don't know if it's related to this issue?

But it is certainly running much better than the current master.

**Comment by Mizipzor** [comment]
Im gonna merge this but it would be interesting to see if we can reproduce that bug.

**Pull request merged**